lib/swiss_village_directory/version.rb

```diff
@@ -1,3 +1,3 @@
 module SwissVillageDirectory
-  VERSION = '0.4.0'.freeze
+  VERSION = '0.5.0'.freeze
```

Comment:
since it's a major change, i'm fine with making it `1.0.0`?

Reply:
I am as well 👍